lighthouse-core/gather/connections/connection.js

```diff
@@ -100,7 +100,8 @@ class Connection {
       // handleRawError returns or throws synchronously; wrap to put into promise chain.
       return callback.resolve(Promise.resolve().then(_ => {
         if (object.error) {
-          return this.handleRawError(object.error, callback.method);
+          log.formatProtocol('method <= browser ERR', {method: callback.method}, 'error');
+          throw new Error(`Protocol error (${callback.method}): ${object.error.message}`);
```

Comment:
Am I right that some domains throw and error if you call disable twice and some do not? Might be worth bringing up with the DT times. That seems inconsistent at best.